include/af/graphics.h

```diff
@@ -312,6 +362,26 @@ AFAPI af_err af_draw_plot(const af_window wind, const af_array X, const af_array
 AFAPI af_err af_draw_hist(const af_window wind, const af_array X, const double minval, const double maxval, const af_cell* const props);
 
 /**
+   C Interface wrapper for drawing arrayis as a surface
+
+   \param[in]   wind is the window handle
+   \param[in]   X is an \ref af_array with the x-axis data points
+   \param[in]   Y is an \ref af_array with the y-axis data points
```

Comment:
Variable names for first two parameters in`af_draw_surface` are not same as function signature, this will throw errors in doxygen.